ui/src/components/Toolbar/DialogToggleButton.jsx

```diff
@@ -32,6 +32,8 @@ class DialogToggleButton extends React.Component {
         <Dialog
           isOpen={this.state.isOpen}
           toggleDialog={this.toggleDialog}
+          onMouseOver={(event) => event.stopPropagation()}
+          onMouseOut={(event) => event.stopPropagation()}
           {...dialogProps}
```

Comment:
This change shouldn’t be in here. Looks like a solution to #2684 which I accidentally included in a commit on this branch.